internal/config/canary_config.go

```diff
@@ -77,11 +78,15 @@ const (
 	StatusTimeWindowDefault              = 300000
 	DynamicConfigFileDefault             = ""
 	DynamicConfigWatcherIntervalDefault  = 30000
+	ActiveTracingDefault                 = false
 )
 
 type DynamicCanaryConfig struct {
-	SaramaLogEnabled  *bool `json:"saramaLogEnabled"`
-	VerbosityLogLevel *int  `json:"verbosityLogLevel"`
+	SaramaLogEnabled      *bool `json:"saramaLogEnabled"`
+	VerbosityLogLevel     *int  `json:"verbosityLogLevel"`
+	ActivateTracing       *bool `json:"activeTracing"`
+	JaegerExporterTracing *bool `json:"jaegerExporterTracing"` //if true activate jaeger else it is opentelemetry exporter
```

Comment:
Is there a particular reason to explicitly support Jaeger at this level? 

I'm thinking `ActivateTracing` is enough and we could enable the `JaegerExporter` if we detect the Jaeger env vars.

Reply:
you're right !

Reply:
now it is based on env variables definition